curriculum/challenges/english/24-b1-english-for-developers/learn-about-adverbial-phrases/67fe8365a79fb94d93868035.md

```diff
@@ -0,0 +1,104 @@
+---
+id: 67fe8365a79fb94d93868035
+title: Task 14
+challengeType: 19
+dashedName: task-14
+---
+
+<!-- (Audio) Bob: Let's think about reducing redundancy. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What does Bob suggest to help with Sarah's concern?
+
+## --answers--
+
+Bob suggests reducing redundancy.
+
+---
+
+Bob suggests increasing the frequency of meetings.
+
+### --feedback--
+
+Bob does not suggest increasing the number of meetings.
+
+---
+
+Bob suggests adding more team members.
+
+### --feedback--
+
+This is not part of Bob's suggestion.
+
+---
+
+Bob suggests focusing on project deadlines.
+
+### --feedback--
+
+Bob's suggestion is not about focusing on deadlines.
+
+## --video-solution--
+
+1
+
+# --explanation--
+
+`Let's think about...` is used to suggest considering an idea or taking some time to reflect on a possible action. When using `about`, the verb that follows it is in the `-ing` form. This is because `about` is a preposition, and after a preposition, you use the `-ing` form of the verb. For example:
```

Comment:
```suggestion
`Let's think about...` is used to suggest considering an idea or taking some time to reflect on a possible action.

When using `about`, the verb that follows it is in the `-ing` form. This is because `about` is a preposition, and after a preposition, you use the `-ing` form of the verb. For example:
```